projects/observability/src/shared/components/utils/chart-tooltip/chart-tooltip-popover.ts

```diff
@@ -30,8 +33,8 @@ export class ChartTooltipPopover<TData, TContext> implements ChartTooltipRef<TDa
       this.popoverRef.updatePositionStrategy({
         type: PopoverPositionType.FollowMouse,
         boundingElement: element,
-        offsetX: 20,
-        offsetY: 30
+        offsetX: 0,
```

Comment:
What would be the impact of this change at other places?

Reply:
will be changing the way that we display the tooltip keeping it visible when the user has the cursor on the tooltip but will not break the functionality.